packages/e2e-test-utils/src/is-in-default-block.js

```diff
@@ -0,0 +1,10 @@
+export function isInDefaultBlock() {
```

Comment:
We should document each of these utility functions, at least in a JSDoc. It wasn't immediately obvious to me precisely what this was testing.